Documentation/ContentElements/Content/Index.rst

```diff
@@ -1,74 +1,198 @@
+    *  Use the pencil icon to edit the element.
+    *  Enable/disable the element to show or hide it in the frontend
+    *  Delete the element.
 
-#. This group of icons is enabled when hovering over the whole element.
+#.  If the edit icons are missing you do not have sufficient permission to
+    edit the content element in question.
+
+..  _content-working-hiding:
+
+Hiding content elements
+=======================
 
-   *  Use the pencil icon to edit the element.
-   *  Enable/disable the element to show or hide it in the frontend
-   *  Delete the element.
+You can temporarily hide content that should be hidden for a while but will be
+needed later. You can also hide a content element while you are still working
+on it. Hidden content elements are not visible in the frontend
 
-.. _content-working-moving:
+..  figure:: /Images/ManualScreenshots/ContentElements/HideContent.png
+    :alt: A content element in the page module is hidden
+
+    Use the "hide" button to hide a content elements. The second content element is hidden and can be unhidden by the same button.
+
+A content element can also be hidden in the "Access" tab of its edit form, its
+context menu or the `List module <https://docs.typo3.org/permalink/t3editors:using-the-list-module-effectively>`_.
+
+..  _content-working-moving:
 
 Moving content elements
 =======================
 
-You can move content elements using drag and drop.
+You can move content elements using drag and drop. You can either drop them
+in one of the highlighted areas on the current page or drop them on a different
+page within the page tree.
+
+..  figure:: /Images/ManualScreenshots/ContentElements/MoveContentElement.png
+    :alt: Screenshot of moving a content element via drag and drop in the TYPO3 backend
+
+    Drop the content element on a page in the page tree to move it to another page
+
+..  _content-working-copy-paste:
+
+Copy and page content elements
+==============================
+
+..  figure:: /Images/ManualScreenshots/ContentElements/CopyContent.png
+    :alt: The context menu of a content menu in the TYO3 backend
+
+    Use the context menu to copy a content element
+
+..  _content-working-life-search:
+
+Finding content using the Live Search
+=====================================
+
+Use the search button on the top left of the TYPO3 backend or the keyboard
+shortcut :kbd:`Ctrl + K` to open the life search.
+
+..  figure:: /Images/ManualScreenshots/ContentElements/LiveSearch.png
+    :alt: Using the live search to search for content elements
+
+    Use the arrow to get more information on the search result
+
+You can then edit the content element containing the text you were searching for.
+
+..  figure:: /Images/ManualScreenshots/ContentElements/PasteContent.png
+    :alt: Location of the paste button beside the "Create new content" button
+
+    Paste the content into the desired location.
+
+You can also copy content elements by pressing :kbd:`Ctrl` during drag and drop.
```

Comment:
Hui. We are in section "search". Seems that part should be moved to copy section

Reply:
Seems like I accidently moved something, will have another look at this PR, thanks for catching it

Reply:
Moved it back in the copy chapter